src/components/Quote.vue

```diff
@@ -145,6 +150,10 @@ export default {
 			return displayName
 		},
 
+		isOwnMessageQuoted() {
+			return this.actorId === this.$store.getters.getUserId()
```

Comment:
```suggestion
			return this.actorId === this.$store.getters.getActorId()
```
Then it also should work for guests

Reply:
just tried with guests now, the actor type "guests" match but not the actor id:

in my test with a guest, this.actorId is "95e4a466030ba6a55d521429f5961e12a5397d07"
and this.$store.getters.getActorId() is "guest/5ddeb308573914adfda3d870c8beb6ac4dbf566b"

Reply:
ohh, maybe it's because I refreshed the page, so the actor id would change...

in any case I think I need to split the "/" there

Reply:
https://github.com/nextcloud/spreed/pull/4940/commits/366464cd419bf0433a64e1b555cd3e26660ecb02